reporting/sndbox.py

```diff
@@ -141,6 +141,9 @@ def run(self, results):
         debug = results['debug']
         custom = json.loads(results['info']['custom'])
         sample = custom["sample"]
+        is_office_package = self.task.get("package") in ['xls', 'doc', 'ppt']
+        logger.info("is office package: %s", is_office_package)
```

Comment:
can this be removed?

Reply:
logging?

Reply:
what's the added value?
a more useful log would be `ignored a detected crash, package is office`